packaging/fedora/arduino-mk.spec

```diff
@@ -25,18 +24,21 @@ Arduino platform.
 %setup -q
 
 %install
-mkdir -p %{buildroot}/%{_datadir}/arduino
-mkdir -p %{buildroot}/%{_bindir}
-mkdir -p %{buildroot}/%{_mandir}/man1
-mkdir -p %{buildroot}/%{_docdir}/%{name}/examples
-install -m 755 -d %{buildroot}/%{_docdir}/%{name}
-install -m 755 -d %{buildroot}/%{_docdir}/%{name}/examples
-for dir in `find examples -type d` ; do install -m 755 -d %{buildroot}/%{_docdir}/%{name}/$dir ; done
-for file in `find examples -type f ! -name .gitignore` ; do install -m 644 $file %{buildroot}/%{_docdir}/%{name}/$file ; done
-install -m 644 *.mk arduino-mk-vars.md %{buildroot}/%{_datadir}/arduino
-install -m 644 licence.txt %{buildroot}/%{_docdir}/%{name}
-install -m 755 bin/ard-reset-arduino %{buildroot}/%{_bindir}/ard-reset-arduino
-install -m 644 ard-reset-arduino.1 %{buildroot}/%{_mandir}/man1
+mkdir -p %{buildroot}%{_datadir}/arduino
+mkdir -p %{buildroot}%{_bindir}
+mkdir -p %{buildroot}%{_mandir}/man1
+mkdir -p %{buildroot}%{_pkgdocdir}/examples
+for dir in `find examples -type d` ; do install -m 755 -d %{buildroot}%{_pkgdocdir}/$dir ; done
+for file in `find examples -type f ! -name .gitignore` ; do install -m 644 $file %{buildroot}%{_pkgdocdir}/$file ; done
+install -m 644 *.mk arduino-mk-vars.md %{buildroot}%{_datadir}/arduino
+install -m 644 licence.txt %{buildroot}%{_pkgdocdir}
+install -m 755 bin/ard-reset-arduino %{buildroot}%{_bindir}/ard-reset-arduino
+install -m 644 ard-reset-arduino.1 %{buildroot}%{_mandir}/man1
+
+# Fedora uses ccache
+sed -i -e 's,\(CC \+= \)\($(AVR_TOOLS_PATH)\)\(/$(CC_NAME)\),\1/usr/lib64/ccache\3,' \
```

Comment:
Hi, you are correct both about the `ccache` lines, and `%{_pkgdocdir}`.  Since F17 (last version without `%{_pkgdocdir}`) was EOL'ed two years ago, it should be fine I think.

About the slashes, I'm pretty sure there should not be any.  But I'll double check nonetheless and report back.

Thanks for reviewing the changes. :)